runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java

```diff
@@ -119,8 +119,8 @@
 	public static final int MINIGAME_TAB_ID = 76;
 	public static final int SPELLBOOK_GROUP_ID = 218;
 	public static final int PVP_GROUP_ID = 90;
+	public static final int FISHING_TRAWLER_GROUP_ID = 366;
 	public static final int ZEAH_MESS_HALL_GROUP_ID = 235;
```

Comment:
My bad, I think my "Suggested change" may have gotten rid of the line; let's try this:
```suggestion
	public static final int ZEAH_MESS_HALL_GROUP_ID = 235;

```

Reply:
No problem. I probably shouldn't be lying in bed reading my emails and accepting suggestions :)